setup.py

```diff
@@ -0,0 +1,71 @@
+import codecs
+import io
+import os
+import re
+
+from setuptools import setup
+
+
+def get_version():
+    with codecs.open(
```

Comment:
Can You copy everything from https://github.com/wikibusiness/aioelasticsearch/blob/master/setup.py#L8 ?